Setup.bat

```diff
@@ -23,7 +23,7 @@ call :MarkStartOfBlock "Setup the git hooks"
     echo check_run() {>>.git\hooks\post-merge
     echo echo "$changed_files" ^| grep --quiet "$1" ^&^& exec $2>>.git\hooks\post-merge
     echo }>>.git\hooks\post-merge
-    echo check_run RequireSetup "cmd.exe /c Setup.bat">>.git\hooks\post-merge
+    echo check_run RequireSetup "cmd.exe /c Setup.bat %*">>.git\hooks\post-merge

```

Comment:
Isn't there a `Setup.sh` we need to consider too?

Reply:
Yep, I also looked at that, but there we just copy `post-merge` from Extras, and I'm not sure if we need to do something special there. I'd say it could be done separately.